ui/apps/platform/src/Containers/Navigation/NavigationSideBar/NavigationSideBar.tsx

```diff
@@ -4,7 +4,7 @@ import { Nav, NavList, NavExpandable, PageSidebar } from '@patternfly/react-core
 
 import { IsFeatureFlagEnabled } from 'hooks/useFeatureFlags';
 import { HasReadAccess } from 'hooks/usePermissions';
-import { knownBackendFlags } from 'utils/featureFlags';
+// import { knownBackendFlags } from 'utils/featureFlags';
```

Comment:
If we are leaving this in for future use, let's add a comment above for why it is needed sometimes.